Pds4File/tests/test_pds4file_blackbox.py

```diff
@@ -31,9 +51,293 @@ def test_opus_id(self, input_path, expected):
         [
             ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm',
              f'{PDS4_HOLDINGS_DIR}/uranus_occs_earthbased/uranus_occ_u0_kao_91cm'),
-        ]
+             ('cassini_iss/cassini_iss_cruise',
+             f'{PDS4_HOLDINGS_DIR}/cassini_iss/cassini_iss_cruise'),
+             ('cassini_vims/cassini_vims_cruise',
+             f'{PDS4_HOLDINGS_DIR}/cassini_vims/cassini_vims_cruise'),
+      ]
     )
     def test_abspath(self, input_path, expected):
         target_pdsfile = instantiate_target_pdsfile(input_path)
         res = target_pdsfile.abspath
         assert res == expected
+
+    @pytest.mark.parametrize(
+        'input_path,expected',
+        [
+            ('uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_alpha_egress_1000m.xml',
+             'bundles/uranus_occs_earthbased/uranus_occ_u0_kao_91cm/data/rings/u0_kao_91cm_734nm_radius_alpha_egress_1000m.xml'),
+             ('cassini_iss/cassini_iss_cruise/data_raw/130xxxxxxx/13089xxxxx/1308947228n.img',
+             'bundles/cassini_iss/cassini_iss_cruise/data_raw/130xxxxxxx/13089xxxxx/1308947228n.img'),
+             ('cassini_vims/cassini_vims_cruise/data_raw/130xxxxxxx/13089xxxxx/1308946681_xxx/1308946681_002.qub',
+             'bundles/cassini_vims/cassini_vims_cruise/data_raw/130xxxxxxx/13089xxxxx/1308946681_xxx/1308946681_002.qub'),
+             ('cassini_vims/cassini_vims_cruise/data_raw/130xxxxxxx/13089xxxxx/1308947235.qub',
+             'bundles/cassini_vims/cassini_vims_cruise/data_raw/130xxxxxxx/13089xxxxx/1308947235.qub'),
+     ]
```

Comment:
Let's fix the formatting/indentation at line 68-75, stick to the code style that one tab = 4 spaces
Same for line 42,58,76,96,132,190,212,264,300,337

Reply:
I've adjusted indentations and commented out uranus occs. I also removed the `_vis` placeholder text for the COVIMS rule